ondewo/nlu/intent.proto

```diff
@@ -209,13 +209,16 @@ message Intent {
         // Required. The text of the training phrase.
         string text = 3;
 
+        // Optional. The language of the training phrase.
+        string language_code = 4;
+
         // Optional. The collection of annotated entities in the training phrase.
-        repeated Entity entities = 4;
+        repeated Entity entities = 5;
```

Comment:
careful!
changing these numbers is not as innocent as it might seem.

In protobuf, to maintain retro-compatibility when adding fields, you just add a new field and associate a new number to it. These numbers are used in serialization. Therefore, if you change a number is like changing the name of the field, you create a non-retro-compatible change.

Reply:
Naturally, if we make a non-retrocompatible change, the version of our API must change on its `major`, not its `minor`.